testdata/p4_16_samples_outputs/p416-type-use3.p4.p4info.txt

```diff
@@ -0,0 +1,261 @@
+pkg_info {
+  arch: "v1model"
+}
+tables {
+  preamble {
+    id: 33586815
+    name: "ingress.custom_table"
+    alias: "custom_table"
+  }
+  match_fields {
+    id: 1
+    name: "hdr.ethernet.srcAddr"
+    bitwidth: 48
+    match_type: EXACT
+    type_name {
+      name: "EthT_t"
+    }
+  }
+  match_fields {
+    id: 2
+    name: "hdr.ethernet.dstAddr"
+    bitwidth: 48
+    match_type: EXACT
+  }
+  match_fields {
+    id: 3
+    name: "hdr.custom.e"
+    bitwidth: 8
+    match_type: EXACT
+  }
+  match_fields {
+    id: 4
+    name: "hdr.custom.ed"
+    bitwidth: 8
+    match_type: EXACT
+  }
+  match_fields {
+    id: 5
+    name: "hdr.custom.et"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomT_t"
+    }
+  }
+  match_fields {
+    id: 6
+    name: "hdr.custom.edd"
+    bitwidth: 8
+    match_type: EXACT
+  }
+  match_fields {
+    id: 7
+    name: "hdr.custom.eddt"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomDDT_t"
+    }
+  }
+  match_fields {
+    id: 8
+    name: "hdr.custom.edtd"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomDT_t"
+    }
+  }
+  match_fields {
+    id: 9
+    name: "hdr.custom.edtt"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomDTT_t"
+    }
+  }
+  match_fields {
+    id: 10
+    name: "hdr.custom.etdd"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomT_t"
+    }
+  }
+  match_fields {
+    id: 11
+    name: "hdr.custom.etdt"
+    bitwidth: 8
+    match_type: EXACT
+    type_name {
+      name: "CustomT_t"
```

Comment:
Since you asked about feedback on the type_name values in this output file, I believe this one should be "CustomTDT_t", not "CustomT_t".

Reply:
Ok, thanks.  This was precisely, one name I wanted to confirm.   I will make code changes, test, and get back.  
@antonin You are off the hook for verifying name since Andy relied.  

This function also happens to be the least pending issue to close after which I can close this PR and issue a new one with one consolidated diffs for final review.